src/Currency/ArgentinePeso.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace PCF\ValueObject\Currency;
+
+class ArgentinePeso extends AbstractCurrency
+{
+
+    /**
+     * AbstractCurrency constructor.
+     *
+     * @param float $quality
+     */
+    public function __construct($quality)
+    {
+        parent::__construct($quality);
+        $this->alphabeticCode = 'ARS';
+        $this->numericCode = '32';
```

Comment:
And i propose add unicode symbol code: http://www.fileformat.info/info/unicode/category/Sc/list.htm